finish/backendServices/src/main/java/io/openliberty/guides/event/dao/EventDao.java

```diff
@@ -34,34 +34,39 @@ public void createEvent(Event event) {
         em.persist(event);
         // end::Persist[]
     }
+    
     // end::createEvent[]
```

Comment:
- hotpsot is highlighting one extra line (for createEvent, etc.):
    ![image](https://user-images.githubusercontent.com/46586499/66317332-4e48e400-e8e7-11e9-8c0b-e39e5714a016.png)

should be:
```
}
// end::createEvent[]

// tag::readEvent[]
```

instead of:
```
}

// end::createEvent[]
// tag::readEvent[]
```